account_analytic_report/models/account_group.py

```diff
@@ -0,0 +1,67 @@
+# ?? 2018 Forest and Biomass Romania SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountGroup(models.Model):
+    _inherit = "account.group"
+
+    group_child_ids = fields.One2many(
```

Comment:
This seems too much logic for just a report, and a bit disconnected from analytic reporting goal.